src/api/server/authn.rst

```diff
@@ -345,3 +345,124 @@ headers to CouchDB with related requests:
 
 Note that you don't need to request :ref:`session <api/auth/session>`
 to be authenticated by this method if all required HTTP headers are provided.
+
+.. _api/auth/jwt:
+
+JWT Authentication
+====================
+
+.. note::
+    To use this authentication method make sure that the
```

Comment:
[grammar nit]

```suggestion
    To use this authentication method, make sure that the
```

Reply:
I don't agree that this is good grammar.

Reply:
A subordinate clause before a main clause should be followed by a comma.  If you reverse the order, then no comma would be correct.

In either case, not a big deal.